pkg/controller/route/route_controller.go

```diff
@@ -69,6 +69,37 @@ type RouteController struct {
 	recorder         record.EventRecorder
 }
 
+// routesState contains a map of podCIDRs and their states belonging to each node
+type routesState struct {
```

Comment:
I think this might belong to a utility package

Reply:
Not sure I agree yet, it's pretty specific to this controller